### DISCUSSION TITLE
Straight lines and shapes

### DISCUSSION THREAD

**Discussion by Mata2002** [geovens/gInk#151] [Ideas]
if you could add some features like zoomit for e.g,,that we could draw straight lines and draw some frequent shapes,it would make it much better

**Comment by pubpub-zz** [comment]
@Mata2002 
you maybe interested in my fork : 
https://github.com/pubpub-zz/ppInk

**Reply by pubpub-zz** [reply]
@Mata2002  Do not hesitate to star ppInk if you like it 😉

**Comment by alamin5G** [comment]
Please, add this feature!
